recipes/pbc/all/conanfile.py

```diff
@@ -0,0 +1,106 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from six import StringIO
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class PbcConan(ConanFile):
+    name = "pbc"
```

Comment:
It's okay using this name. In the future we will need to consider https://github.com/cloudwu/pbc too.

Reply:
I'm open to naming it something longer, as I don't expect this to be a very commonly used package. How about either pair_crypto or pairing_based_crypto?

Reply:
You can keep `pbc`. Conan is not the only package manager using this same name for this project.

Reply:
Ok, thanks!